src/pch.h

```diff
@@ -52,6 +52,11 @@
 #include <stack>
 #include <unordered_map>
 
+#pragma warning(push)
+// Look at this file if you want to know what they are and what they mean
+#include "pch_cppcorecheck_suppressions.h"
+#include "pch_microsoft_suppressions.h"
```

Comment:
These have the suppression in greater detail in case someone doesn't know what they are.